lib/src/widgets/bottom_bar_with_sheet_item.dart

```diff
@@ -95,7 +91,7 @@ class BottomBarWithSheetItem extends StatelessWidget {
   }
 
   bool _checkItemState(BottomBarBloc barBloc) {
-    return _index == barBloc.selectedIndex;
+    return noSelectionState ? false :  (_index == barBloc.selectedIndex);
```

Comment:
I am looking at your suggestion and I not sure it's correct.
If noSelectionState is true this function should ALWAYS return false, no matter what the index, because we are there is no button state, the buttons are effectively push buttons. ( so we don't ever want them styled).

It seems like 
  return noSelectionState && (_index == barBloc.selectedIndex);

would return true when noSelectionState is true and the index matches the selected index... If noSelectionState is true we again.. NEVER want it to return true, there is no 'selected' button.